.github/workflows/release-typescript-functions.yaml

```diff
@@ -1,9 +1,9 @@
-name: release-kpt-functions
+name: release-typescript-functions
 
 on:
   push:
     tags:
-      - release-kpt-functions-*
+      - release-typescript-functions-*
```

Comment:
This will impact the release process. https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/CONTRIBUTING.md#releases

Reply:
Good catch!  Thank you.